lib/include/tick/prox/prox_l1w.h

```diff
@@ -49,6 +56,22 @@ class DLL_PUBLIC TProxL1w : public TProxSeparable<T> {
 
   void set_weights(SArrayTPtr weights) { this->weights = weights; }
 
+  template <class Archive>
+  void serialize(Archive &ar) {
+    ar(cereal::make_nvp("ProxSeparable",
+                        cereal::base_class<TProxSeparable<T>>(this)));
+
+    ar(CEREAL_NVP(*weights));
```

Comment:
Does this work this simply? I would have thought it would have been necessary to do something similar to:
https://github.com/Dekken/tick/blob/master/lib/include/tick/base_model/model_labels_features.h#L64

Reply:
I recall that, and perhaps, the gtest seems ok so perhaps we can validate from that?